CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.2.0
```

Comment:
4.1.1 right, since it isn't adding anything or removing anything?

Reply:
Erp, good catch. Done.